modules/order/src/AdjustableInterface.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\commerce_order;
+
+/**
+ * Defines an interface for objects that interact with Adjustments.
+ *
+ * @todo Provide a trait? But that'd assume the field is always 'adjustments'
```

Comment:
@bojanz thoughts? If we add commerce_pricing, we can put the interface an an AdjustableTrait, cuts down copy pasta a bit. Especially once we have shipments.